### DISCUSSION TITLE
[#1040910] Images in Buttons

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#112] [Feedback]
OreUI already allows you to add images to buttons so why not add that to Editor mode too!

![2B9D62C8-AB4F-4AE4-A818-221E0390CC6B](https://user-images.githubusercontent.com/47607185/235218795-85f4a20b-ad1a-4bb4-a31b-57ab7fa17eda.png)

**Comment by minecraftchloe** [comment]
YES! Fun fact: The OreUI components used for Editor are custom versions from what is used in Vanilla...for _reasons_. We can add this to the list of things to expose in our Editor API for Editor components!

**Reply by xKingDark** [reply]
Thank you Chloe!

**Comment by m-brisch** [comment]
@DarkGamerYT you should be able to do this now with the new `addButton` API by specifying the `icon` parameter!
![image](https://github.com/user-attachments/assets/e10487fd-d589-4d97-9973-4d82b877ba48)
Happy Editing 😄
